data-serving/data-service/src/controllers/case.ts

```diff
@@ -424,3 +424,35 @@ export const listPlacesOfTransmission = async (
         return;
     }
 };
+
+/**
+ * List most frequently used occupations.
+ *
+ * Handles HTTP GET /api/cases/occupations.
+ */
+export const listOccupations = async (
+    req: Request,
+    res: Response,
+): Promise<void> => {
+    const limit = Number(req.query.limit) || 5;
+    if (limit < 1) {
```

Comment:
Likewise, isn't this impossible with the OpenAPI spec?

Reply:
Removed